baseapp/baseapp.go

```diff
@@ -137,21 +136,21 @@ func (app *BaseApp) MountStore(key sdk.StoreKey, typ sdk.StoreType) {
 }
 
 // load latest application version
-func (app *BaseApp) LoadLatestVersion(mainKey sdk.StoreKey) error {
+func (app *BaseApp) LoadLatestVersion(mainKey *sdk.KVStoreKey) error {
 	err := app.cms.LoadLatestVersion()
 	if err != nil {
 		return err
 	}
-	return app.initFromStore(mainKey)
+	return app.initFromMainStore(mainKey)
 }
 
 // load application version
-func (app *BaseApp) LoadVersion(version int64, mainKey sdk.StoreKey) error {
+func (app *BaseApp) LoadVersion(version int64, mainKey *sdk.KVStoreKey) error {
```

Comment:
I think we should update this godoc to state that it should only be called once (since `initFromMainStore` will panic otherwise).

Reply:
yeah we need better documentation of this function - let me what you think of my additions